pkg/routeagent-driver/main.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"flag"
+	"os"
+
+	"github.com/kelseyhightower/envconfig"
+	"k8s.io/klog"
+	"sigs.k8s.io/controller-runtime/pkg/manager/signals"
+
+	"github.com/submariner-io/submariner/pkg/event"
+	"github.com/submariner-io/submariner/pkg/event/controller"
+	"github.com/submariner-io/submariner/pkg/event/logger"
+	kp "github.com/submariner-io/submariner/pkg/routeagent-driver/handlers/kubeproxy-iptables"
```

Comment:
kp -> kp_iptables (so the code later is more understandable in the kp.NewHandler)

Reply:
Ok